src/view/home.js

```diff
@@ -41,21 +41,26 @@ const HomeView = ({
   transaction,
   nav,
 }) => {
-  const { depositLabel, channelBalanceLabel, unitLabel } = store;
+  const {
+    totalBalanceLabel,
+    unitLabel,
+    channelStatus,
+    channelPercentageLabel,
+  } = store;
   return (
     <Background image="purple-gradient-bg">
       <HomeHeader
-        goChannels={() => channel.init()}
+        goDeposit={() => nav.goDeposit()}
         goSettings={() => nav.goSettings()}
-        showChannelAlert={store.showChannelAlert}
       />
-      <QrCodeSeparator goDeposit={() => nav.goDeposit()} />
       <MainContent style={styles.content}>
         <BalanceDisplay
-          depositLabel={depositLabel}
-          channelBalanceLabel={channelBalanceLabel}
+          totalBalanceLabel={totalBalanceLabel}
           unitLabel={unitLabel}
+          channelStatus={channelStatus}
+          channelPercentageLabel={channelPercentageLabel}
```

Comment:
Totally love the way this looks! Way clearer than any of the other balance ideas we had.